spec/support/shared_examples/validators/shared_examples_for_advocate_claim_validators.rb

```diff
@@ -164,6 +172,33 @@
       end
     end
 
+    if ['advocate_claim', 'advocate_hardship_claim'].include? options[:claim_type]
```

Comment:
this conditional logic implies this should not be in the shared set at all i'd say

Reply:
👍 Have removed conditional logic and refactored into shared example 'common defendant basic fees aggregation validation'.